ExampleAppUITests/TradeItIosTicketSDK2UITests.swift

```diff
@@ -3,71 +3,180 @@ import TradeItIosTicketSDK2
 
 class TradeItIosTicketSdk2UITests: XCTestCase {
     var application: XCUIApplication!
-
+    
     override func setUp() {
         super.setUp()
         continueAfterFailure = false
-
+        
         self.application = XCUIApplication()
         self.application.launchArguments.append("isUITesting")
         self.application.launch()
         sleep(1)
     }
-
+    
     override func tearDown() {
         super.tearDown()
     }
-
-    func testWelcomeFlow() {
+    
+    func testSecurityQuestionFlow() {
         let app = self.application
-
+        
         clearData(app)
-
-        handleWelcomeScreen(app)
+        
+        handleWelcomeScreen(app, launchOption: "LaunchPortfolio")
+        
+        selectBrokerFromTheBrokerSelectionScreen(app, longBrokerName: "Dummy Broker")
+        
+        submitValidCredentialsOnTheLoginScreen(app, longBrokerName: "dummySecurity")
+        
+        waitForElementToAppear(app.alerts["Security Question"])
+        
+        let securityQuestionAlert = app.alerts["Security Question"]
+        securityQuestionAlert.textFields[""].typeText("tradingticket")
+        securityQuestionAlert.buttons["Submit"].tap()
+        
+        waitForElementToAppear(app.navigationBars["Portfolio"])
```

Comment:
Can we add a wrong answer here and then put in the correct answer? The security question handler is doing something pretty funky so I want to make sure it works on the wrong answer case as well.